src/examples/libpmem2/map_multiple_files/map_multiple_files.c

```diff
@@ -0,0 +1,280 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2020, Intel Corporation */
+
+/*
+ * map_multiple_files.c -- implementation of virtual address allocation example
+ */
+
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#ifndef _WIN32
+#include <unistd.h>
+#else
+#include <io.h>
+#endif
+#include <libpmem2.h>
+
+/*
+ * cleanup_arrays - free dynamically allocated memory
+ */
+static void
+free_arrays(size_t **file_size, int **fd, struct pmem2_map ***map,
```

Comment:
The subject should usually precede the verb, so instead of `free_arrays` use `arrays_free`. Same everywhere.